.jenkins/pipelines/pr.Jenkinsfile

```diff
@@ -3,11 +3,9 @@ APPROVED_AUTHORS = [
     'asvrada',
     'bodzhang',
     'CyanDevs',
-    'Francis-Liu',
```

Comment:
Maybe remove Xuejun too?

Reply:
Xuejun is still at Microsoft - @radhikaj thoughts?